src/router.js

```diff
@@ -20,5 +21,6 @@ router.use('/blockchains', blockchains);
 router.use('/tokens', tokens);
 router.use('/admins', admins);
 router.use('/view-blockchains', view_blockchains);
+router.use('/webhook', webhook);
```

Comment:
@tcoratger What do you think about these 2 approaches?

1. Use a `/webhook` route and keep all changes inside this route with different endpoints, for example:
- [PUT] `/webhook/offer/status`
-  [PUT] `/webhook/offer/max-price-limit`

2. Use the already created routes and different endpoints to update what comes from the zapier integration, for example:
- [PUT] `/offers/status`
- [PUT] `/offers/max-price-limit`

3. Use the already created routes and the actual put method to update what comes from the zapier integration, for example:
- [PUT] `/offers`
In this example, we can use the validators with the `optional` flag to validate every change.

Do you see any new way to approach the listeners?

Reply:
@juniormp I think you can open a dedicated `/webhooks` route here as it is a specific feature we are implementing here, adding it to already existing routes could bring confusion, what do you think?

Reply:
@tcoratger I agree I think its better to avoid any further problems